### PR TITLE
feat: Multi licenses formatting

### PR DESCRIPTION
**Pull request by lili2311**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?
Add formatting for licenses new data shape and add some extra styles for multi licenses

##### Remediation based format
Screen Shot 2019-10-03 at 14 32 15

##### Non-remediation format
Screen Shot 2019-10-03 at 14 31 07

####### remediation based
Screen Shot 2019-10-03 at 14 32 45
####### non-remediation
Screen Shot 2019-10-03 at 14 38 41

Relies on https://github.com/snyk/registry/pull/9720/files

### CODE REVIEW

File: src/cli/commands/test/formatters/legal-license-instructions.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import * as wrap from 'wrap-ansi';
+import chalk from 'chalk';
+
+export function formatLegalInstructions(
+  legalInstructions: LegalInstruction[],
+): string {
+  const legalContent: string[] = legalInstructions.map((legalData) =>
+    wrap(chalk.bold(`○ for ${legalData.licenseName}: `) + legalData.legalContent, 100).split('\n').join('\n    '),
+  );
+  return legalContent.join('\n    ').split('\n').join('\n');
```
Comment (orsagie) [inline src/cli/commands/test/formatters/legal-license-instructions.ts]: why are you spliting and joining on the same `\n`?

File: src/cli/commands/test/formatters/remediation-based-format-issues.ts [hunk 2]
```diff
@@ -23,7 +25,7 @@ interface BasicVulnInfo {
   name: string;
   version: string;
   fixedIn: string[];
-  legalInstructions?: string;
+  legalInstructionsArray?: LegalInstruction[];
```
Comment (orsagie) [inline src/cli/commands/test/formatters/remediation-based-format-issues.ts]: Can you change back to `legalInstructions`. Seeing as there is a single LegalInstruction interface - having a plural named parameter is helpful enough in knowing it's an array.

File: src/cli/commands/test/index.ts [hunk 3]
```diff
@@ -587,6 +587,7 @@ function groupVulnerabilities(vulns): GroupedVuln[] {
       map[curr.id].dockerBaseImage = curr.dockerBaseImage;
       map[curr.id].nearestFixedInVersion = curr.nearestFixedInVersion;
       map[curr.id].legalInstructions = curr.legalInstructions;
```
Comment (orsagie) [inline src/cli/commands/test/index.ts]: do we still need this line?

File: src/cli/commands/test/formatters/legal-license-instructions.ts (hunk 1 above)
Comment (lili2311) [reply src/cli/commands/test/formatters/legal-license-instructions.ts]: aha this might be my bug ty

File: src/cli/commands/test/index.ts (hunk 3 above)
Comment (lili2311) [reply src/cli/commands/test/index.ts]: nope

**Pull request merged**